validation/setup_env.sh

```diff
@@ -17,15 +17,16 @@ source "${CONDA_BASE}/etc/profile.d/conda.sh"
 conda env create -f conda_env_validation.yml
 conda activate three-gen-validation
 conda info --env
+
 CUDA_HOME=${CONDA_PREFIX}
 
 echo -e "\n\n[INFO] Installing diff-gaussian-rasterization package\n"
-mkdir -p ./extras/diff_gaussian_rasterization/third_party
-git clone --branch 0.9.9.0 https://github.com/g-truc/glm.git ./extras/diff_gaussian_rasterization/third_party/glm
-pip install ./extras/diff_gaussian_rasterization
+mkdir -p ./extras/diff-gaussian-rasterization
+git clone https://github.com/ashawkey/diff-gaussian-rasterization ./extras/diff-gaussian-rasterization
```

Comment:
do we have to pin the version?

Reply:
they do not have a specific version and I have a feeling the project is sort of frozen (last update 7 months ago)